README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
```

Comment:
I've seen this comment and had similar feedback once or twice before.

While I understand that the way I worded it may sound harsh a bit, but I don't fully agree with the comment.
"Use the source" is not a way to say: hey, reverse engineer my code. Usage of impress.js in its basic form is writing HTML, and demo HTML shows all needed classes to use with comments explaining it.

And the comparison to the car is not very accurate. As a user of a car you don't need to know how it works. You obviously don't need schematics. But impress.js is not just something you 'use' - you develop your code with it. It's a tool you use to build your own 'car' - so why not having schematics for it? ... but that's just a side note ;)

And during this years I received much more positive feedback about comments in impress.js then negative so I'd keep the "use the source" link, Yoda in the demo and in source code.

But because of your hard work @FagnerMartinsBrack we have some docs now, so we should link to them here as well.